axelrod/strategies/lookerup.py

```diff
@@ -165,23 +167,310 @@ def strategy(self, opponent):
         return action
 
 
-# Create several classes at runtime based on the data in the look up tables
-# loaded above, one for each key.
-
-for k, (initial, pattern) in patterns.items():
-    name, plays, op_plays, op_start_plays = k
-    table = create_lookup_table_from_pattern(
-        plays, op_plays, op_start_plays, pattern)
-    class_name = "EvolvedLookerUp{}{}_{}_{}".format(
-        name, plays, op_plays, op_start_plays)
-    # Dynamically create the class
-    new_class = type(class_name, (LookerUp,), {})
-    new_class.__init__ = init_args(partial(
-        new_class.__init__, lookup_table=table, initial_actions=initial))
-    new_class.name = class_name
-    # Add the generated class to the module dictionary so it can be
-    # imported into other modules
-    setattr(module, class_name, new_class)
+class EvolvedLookerUp0_0_1(LookerUp):
```

Comment:
Sure, you only include 1-1-1 and 2-2-2 in the whole list. Should we go ahead and just include those?